tests/integration/test_end_to_end_workflow.py

```diff
@@ -173,3 +175,81 @@ def test_simple_fhir_server_workflow(runner: CliRunner, project_id, tmpdir) -> N
     # TODO note, this does not remove the files from the bucket (UChicago bug)
     # See https://ohsucomputationalbio.slack.com/archives/C043HPV0VMY/p1714065633867229
     run(runner, ["--debug", "projects", "empty", "--project_id", project_id, "--confirm", "empty"])
+
+
+def test_push_fails_with_invalid_doc_ref_creation_date(runner: CliRunner, project_id: str, tmp_path: Path):
+    
+    # check
+    assert os.environ.get("G3T_PROFILE"), "G3T_PROFILE environment variable must be set."
+
+    # copy fixture to temp test dir
+    project_dir = "fhir-gdc-examples"
+    fixtures_path = Path(os.path.dirname(__file__)).parent / "fixtures"
+    fhir_gdc_dir = fixtures_path / project_dir
+    modified_doc_ref_path = fixtures_path / "negative-examples/fhir-gdc-DocumentReference-invalid-date.ndjson"
```

Comment:
This file hasn't been committed

Reply:
pushed up in [e2f8ec3](https://github.com/ACED-IDP/gen3_util/pull/106/commits/e2f8ec354cf6b3c3a781684ebae6cfd468172f2d) lmk if you run into anything else